CONTRIBUTING.md

```diff
@@ -14,11 +14,13 @@ We welcome contributions to `Rgemini` by all GEMINI team members. To submit a co
 
 6. If appropriate, add unit tests in the `tests/testthat/` directory.
 
-7. Submit a [pull request](https://help.github.com/articles/using-pull-requests) into the `develop` branch.
+7. If adding new functions with new documentation, run `devtools::document()` and update the `_pkgdown.yml` file by adding the new function to the appropriate reference section.
 
```

Comment:
Should we add here on the addition of data file? When function requires the use of a new dataset (e.g mapping files), these data files should be uploaded to `/data` folder, and a description of the file should be added to `data.R`. 
Anne and I discussed about standardizing the format of data file to `.rda` to allow data to be loaded by `data(file_name, package="Rgemini")`. If you think this would be a good approach for store & use of data files, we should specify the standard data format here as well.